core/src/main/java/org/geysermc/geyser/translator/sound/block/FenceGateSoundInteractionTranslator.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-2022 GeyserMC. http://geysermc.org
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.geyser.translator.sound.block;
+
+import org.cloudburstmc.math.vector.Vector3f;
+import org.geysermc.geyser.GeyserImpl;
+import org.geysermc.geyser.session.GeyserSession;
+import org.geysermc.geyser.translator.sound.BlockSoundInteractionTranslator;
+import org.geysermc.geyser.translator.sound.SoundTranslator;
+import org.geysermc.geyser.util.SoundUtils;
+
+@SoundTranslator(blocks = {"fence_gate"})
+public class FenceGateSoundInteractionTranslator implements BlockSoundInteractionTranslator {
+    @Override
+    public void translate(GeyserSession session, Vector3f position, String identifier) {
+        if (identifier.contains("iron")) return;
```

Comment:
Is this check needed here?

Reply:
No, this was a mistake, I forgot to delete this before opening a pull request, sorry. I'll make a commit now

Reply:
Its now removed